external/dxc/GIT-COMMIT.txt

```diff
@@ -0,0 +1 @@
+7a8c4a20da905a736637595fc6111c3ce033ba21 
```

Comment:
Why is the entirety of dxc in this PR?

Reply:
Probably because we are using the binary version, not sure if we need all of those files tho.

Reply:
Submodule? This is an external repo.

Reply:
I actually have no idea where @DethRaid has this files from

Reply:
https://github.com/project-arcana/dxc-wrapper

Reply:
Fixed